src/base/http/server.cpp

```diff
@@ -54,6 +60,10 @@ Server::Server(IRequestHandler *requestHandler, QObject *parent)
 #ifndef QT_NO_OPENSSL
     QSslSocket::setDefaultCiphers(safeCipherList());
 #endif
+
+    QTimer *dropConnectionTimer = new QTimer(this);
+    connect(dropConnectionTimer, SIGNAL(timeout()), SLOT(dropTimeoutConnections()));
```

Comment:
IMO, it's better to use Qt5 connect syntax in new code since it does compile time checks.